apps/edxapp/templates/_volumes/exports.yml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: edxapp-pvc-exports
```

Comment:
Not sure I would have used the plural form. What about `edxapp-pvc-export` instead?

Reply:
there are many kinds of export types. so tell me.

Reply:
@jmaupetit anyway it seems our OpenShift creates Volumes with a minimum size of 1Go. Are you aware of this?

Reply:
Nope I wasn't.

@rmoch ok for the singular form.